### PR TITLE
Chore: deprecate containerized workload

### LINKED ISSUE #2261: [Chore] deprecate containerized workload and remove the related code
same as #2258

### PR DESCRIPTION
**Pull request by reetasingh**
**What this PR does / why we need it**:
deprecate containerized workload and replace the tests with Deployment
**Which issue(s) this PR fixes**:

Fixes #2261

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by reetasingh** [comment]
> Please run 'make reviewable' to keep consistent format.

Thanks @Somefive , I have added a new commit after running make reviewable

**Review by wonderflow — APPROVED** [review]

**Comment by zzxwill** [comment]
@reetasingh Please search your branch by keyword `containerizedworkload`, here are left a few .go and .yaml files to clean up, like https://github.com/reetasingh/kubevela/blob/cad21c5138b567ca75ff56fae318e72fbf6706e0/pkg/controller/utils/utils.go#L73

File: test/e2e-test/health_scope_test.go [hunk 1]
```diff
@@ -104,58 +104,38 @@ var _ = Describe("HealthScope", func() {
...
-					},
+			Spec: appsv1.DeploymentSpec{
+				Selector: &metav1.LabelSelector{
+					MatchLabels: label,
 				},
-			},
-		}
-		logf.Log.Info("Creating workload definition")
-		// For some reason, WorkloadDefinition is created as a Cluster scope object
-		Expect(k8sClient.Create(ctx, &wd)).Should(SatisfyAny(BeNil(), &util.AlreadyExistMatcher{}))
```
Comment (captainroy-hy) [inline test/e2e-test/health_scope_test.go:133]: Is this the reason why e2e test failed?

**Comment by reetasingh** [comment]
> @reetasingh Please search your branch by keyword `containerizedworkload`, a few .go and .yaml files are left to clean up, like https://github.com/reetasingh/kubevela/blob/cad21c5138b567ca75ff56fae318e72fbf6706e0/pkg/controller/utils/utils.go#L73

@zzxwill I have removed the remaining references.

**Review by zzxwill — APPROVED** [review]
/lgtm

**Review by hongchaodeng — APPROVED** [review]

**Review by captainroy-hy — APPROVED** [review]
LGTM

**Comment by wonderflow** [comment]
COOL

**Pull request merged**